packages/contracts-bedrock/.storage-layout

```diff
@@ -281,11 +281,33 @@
 ➡ contracts/L2/SequencerFeeVault.sol:SequencerFeeVault
 =======================
 
-+---------------+---------+------+--------+-------+------------------------------------------------------+
-| Name          | Type    | Slot | Offset | Bytes | Contract                                             |
-+========================================================================================================+
-| spacer_0_0_20 | address | 0    | 0      | 20    | contracts/L2/SequencerFeeVault.sol:SequencerFeeVault |
```

Comment:
Note that the spacer for the legacy `l1FeeWallet` is in the same place. This value is in the storage of the `SequencerFeeVault` but the contract now uses an immutable to read that value. This allows for the admin of the proxy to upgrade the contract to change the withdrawal account

Reply:
I'm confused by why this is even showing a diff on these lines, can you see what's changed?